src/IIS/Settings/SiteSettings.cs

```diff
@@ -1,13 +1,23 @@
 ﻿#region Using Statements
-    using Cake.Core.IO;
+
+using System;
+using Cake.Core.IO;
+    using Cake.IIS.Settings;
+    using Cake.IIS.Settings.Bindings;
+    using Cake.IIS.Settings.Bindings.FluentAPI;
+
 #endregion
 
 
 
 namespace Cake.IIS
 {
-    public abstract class SiteSettings : BindingSettings, IDirectorySettings
+    public abstract class SiteSettings : IDirectorySettings
     {
+        #region Fields (1)
```

Comment:
Why is this here? Your effectively allowing people to update something via obsolete methods that isn't actually used by the Managers.